server/routes/checkYourAnswers/checkYourAnswers.test.ts

```diff
@@ -3,33 +3,79 @@ import request from 'supertest'
 import appWithAllRoutes from '../testutils/appSetup'
 import PlaceOnReportService from '../../services/placeOnReportService'
 import LocationService from '../../services/locationService'
+import UserService from '../../services/userService'
+import DecisionTreeService from '../../services/decisionTreeService'
 
 jest.mock('../../services/placeOnReportService.ts')
 jest.mock('../../services/locationService.ts')
+jest.mock('../../services/userService.ts')
+jest.mock('../../services/decisionTreeService.ts')
 
 const placeOnReportService = new PlaceOnReportService(null) as jest.Mocked<PlaceOnReportService>
 const locationService = new LocationService(null) as jest.Mocked<LocationService>
+const userService = new UserService(null) as jest.Mocked<UserService>
+const decisionTreeService = new DecisionTreeService(
+  placeOnReportService,
+  userService
+) as jest.Mocked<DecisionTreeService>
 
 let app: Express
 
 beforeEach(() => {
-  app = appWithAllRoutes({ production: false }, { placeOnReportService, locationService })
-  placeOnReportService.getPrisonerDetails.mockResolvedValue({
-    offenderNo: 'G6415GD',
-    firstName: 'UDFSANAYE',
-    lastName: 'AIDETRIA',
-    assignedLivingUnit: {
-      agencyId: 'MDI',
-      locationId: 25928,
-      description: '4-2-001',
-      agencyName: 'Moorland (HMP & YOI)',
+  decisionTreeService.allOffences.mockResolvedValue([
+    {
+      victimOtherPersonsName: undefined,
+      victimPrisonersNumber: 'G6123VU',
+      victimStaffUsername: undefined,
+      offenceCode: '2001',
     },
-    categoryCode: undefined,
-    language: 'English',
-    friendlyName: 'Udfsanaye Aidetria',
-    displayName: 'Aidetria, Udfsanaye',
-    prisonerNumber: 'G6415GD',
-    currentLocation: 'Moorland (HMP & YOI)',
+  ])
+
+  decisionTreeService.adjudicationData.mockResolvedValue({
+    draftAdjudication: {
+      id: 100,
+      prisonerNumber: 'G6415GD',
+      incidentDetails: {
+        locationId: 197682,
+        dateTimeOfIncident: '2021-12-09T10:30:00',
+        handoverDeadline: '2021-12-11T10:30:00',
+      },
+      incidentRole: {
+        roleCode: '25c',
+      },
+      offenceDetails: [
+        {
+          offenceCode: 1,
+          victimPrisonersNumber: 'G5512G',
+        },
+        {
+          offenceCode: 2,
+        },
+      ],
+      incidentStatement: { statement: 'text here', completed: true },
+      startedByUserId: 'TEST_GEN',
+    },
+    adjudicationNumber: 100,
+    // @ts-expect-error: Type '"assisted"' is not assignable to type 'IncidentRole'.
```

Comment:
Any ideas about how to get rid of this TS error? I've tried changing the type to string (which is what comes back!) but it doesn't solve it (as it causes downstream issues)